src/js/themes/base.js

```diff
@@ -346,6 +346,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // margin: undefined
       },
       hover: {
+        // background: undefined,
```

Comment:
My only thought is whether we should consider the lack of hover indicator a bug and add a default here.

Reply:
I don't think I'd consider it a bug because grommet theme has a hover behavior already that changes the heading color of the text.